plugins/pulp_rpm/plugins/importers/yum/sync.py

```diff
@@ -541,9 +553,17 @@ def update_content(self, metadata_files, url):
         :type  metadata_files:  pulp_rpm.plugins.importers.yum.repomd.metadata.MetadataFiles
         :param url: curret URL we should sync
         :type: str
+
+        :raises PulpCodedException: if there is some inconsistency in metadata
         """
         catalog = PackageCatalog(self.conduit.importer_object_id, url)
         rpms_to_download, drpms_to_download = self._decide_what_to_download(metadata_files, catalog)
+
+        if (self.package_count[primary.METADATA_FILE_NAME] !=
+                self.package_count[other.METADATA_FILE_NAME]):
+            reason = 'metadata is missing for some packages in filelists.xml and in other.xml'
```

Comment:
It is done in [one place where error code/message is defined](https://github.com/pulp/pulp_rpm/pull/1006/files#diff-140641dd82dde9606a1467743c1ebcdeR38).